mount_darwin.go

```diff
@@ -24,7 +24,7 @@ func loadOSXFUSE(bin string) error {
 	cmd.Stdout = os.Stdout
 	cmd.Stderr = os.Stderr
 	err := cmd.Run()
-	return err
+	return fmt.Errorf("Error loading fuse (%s): %s", bin, err)
```

Comment:
The downside to this is that it makes it impossible to retrieve the error code that is returned by load_osxfuse.  I'm using this at the moment to detect the condition on 10.13 where the kext load is rejected due to not being approved by the user.